easybuild/scripts/install-EasyBuild-develop.sh

```diff
@@ -69,6 +69,7 @@ conflict    EasyBuild
 
 prepend-path    PATH            "\$root/easybuild-framework"
 
+prepend-path    PYTHONPATH      "\$root/vsc-base"
```

Comment:
You're right, will fix.

I was still testing the updated script myself as well, but bumped into another issue (that I still need to figure out).